server/api/.env.example

```
@@ -27,3 +27,22 @@ GITHUB_APP_ID=
+#   Internal/private URLs must NOT be set here. When unset the alert email
+#   omits the link line and falls back to sentry_issue_id only.
+#
+# Epic #616 Phase 3（issue #809）用の通知設定。
+# - MONITORING_NOTIFY_EMAIL: severity が `high` / `medium` の API エラー通知の
+#   宛先メール。未設定時は notifier が no-op となり、AI コールバックは正常に
+#   動作する（ステージング段階で通知のみ無効化できる）。
+# - ADMIN_BASE_URL: 管理画面の公開オリジン（例: `https://admin.zedi-note.app`）。
+#   通知メール本文に `<base>/errors/<api_error_id>` のリンクを差し込む際に使う。
+#   内部 URL を設定してはいけない。未設定時はリンク行を省略する。
+MONITORING_NOTIFY_EMAIL=
+ADMIN_BASE_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ADMIN_BASE_URL key should go before the MONITORING_NOTIFY_EMAIL key

(UnorderedKey)